### PR TITLE
EPMRPP-35563 React. Log level. Copy/Send defect type modal

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/common/urls.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ export const URLS = {
     `${urlBase}project/${activeProject}/emailconfig`,
   suite: (activeProject, suiteId) => `${urlBase}${activeProject}/item/${suiteId}`,
 
+  testItemsUpdateIssues: (activeProject) => `${urlBase}${activeProject}/item`,
```
Comment (DiscoElevator) [inline app/src/common/urls.js]: Can we use `testItem` url below?

Comment (AmsterGet) [reply app/src/common/urls.js]: I'll try to use it

File: app/src/pages/inside/logsPage/modals/copySendDefectModal/copySendDefectModal.jsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      {
+        issue: {
+          ...itemForCopy.issue,
+          externalSystemIssues: itemForCopy.issue.externalSystemIssues || [],
+        },
+        test_item_id: lastHistoryItem.id,
+      },
+    ];
+
+    fetch(URLS.testItem(activeProject, ''), {
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/modals/copySendDefectModal/copySendDefectModal.jsx]: It's better to rewrite url function like this:
`testItem: (activeProject, id = '') => //...`
and then usage in this case will be:
`URLS.testItem(activeProject)`

Comment (AmsterGet) [reply app/src/pages/inside/logsPage/modals/copySendDefectModal/copySendDefectModal.jsx]: Sorry for my inattention. Fixed

**Pull request merged**